examples/complex/index.js

```diff
@@ -1,56 +1,88 @@
-require('babel-register')
-require('babel-polyfill')
-
 global.artifacts = artifacts;
-global.ContractsProvider = require('../../src/utils/ContractsProvider').default;
+global.ContractsProvider = require('zos-lib/src/utils/ContractsProvider').default;
+const network = process.argv[5];
```

Comment:
Accessing the value of an option by position seems a bit fragile. Maybe use a package to parse the command line arguments, and retrieve network from there? Alternatively, since you are using network just for the stdlib, you can just accept the stdlib address as a parameter or env var, and deploy it if none is specified.